index .html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dr. Norman Borlaug</title>
+  </head>
+  <body>
+    <main id="main">
+      <header>
+        <h1 id="title">Dr. Norman Borlaug</h1>
+      <p>The man who saved a billion lives</p>
+    </header>
+      <div id="img-div">
+        <img
+          id="image" 
+          src="./img/tribute-page-main-image.jpg"
+          alt="dr-norman-and-team"
+        />
+        <div id="img-caption">
+        <p id="tribute-info">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </p>
+    </div>
+      </div>
+    </header>
+      <div class="main-text">
+        <h4>Here's a time line of Dr. Borlaug's life:</h4>
+        <ul class="years">
+          <li><strong>1914</strong> - Born in Cresco, Iowa</li>
+          <li>
+            <strong>1933</strong>- Leaves his family's farm to attend the
+            University of Minnesota, thanks to a Depression era program known as
+            the "National Youth Administration"
+          </li>
+          <li>
+            <strong>1935</strong>- Has to stop school and save up more money.
```

Comment:
Great work 👋😊🙌I think you could bold tag instead of a strong tag. Semantically strong is used to imply importance, urgency, or seriousness. Instead, use the tag or CSS to make the font bold. You can read more about that here point_right https://html.com/tags/strong/